clang/lib/3C/ConstraintVariables.cpp

```diff
@@ -1136,12 +1136,12 @@ void PointerVariableConstraint::constrainOuterTo(Constraints &CS, ConstAtom *C,
         if (*CA < *C) {
           llvm::errs() << "Warning: " << CA->getStr() << " not less than "
                        << C->getStr() << "\n";
-          assert(CA == CS.getWild()); // definitely bogus if not
+          //assert(CA == CS.getWild()); // definitely bogus if not
         }
       } else if (*C < *CA) {
         llvm::errs() << "Warning: " << C->getStr() << " not less than "
                      << CA->getStr() << "\n";
-        assert(CA == CS.getWild()); // definitely bogus if not
+        //assert(CA == CS.getWild()); // definitely bogus if not
```

Comment:
Ah, yes! :)

Reply:
That's the only other place I found with the same type of assertion. I don't think I've run into trouble with this assertions, but it makes sense to disable them at the same time.

Reply:
OK sounds good. I think we are ready to go.